src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -443,3 +445,41 @@ def generate_maps(
     #   }
 
     return maps
+
+
+def subset_results_by_state(processed_results_dir: str, state: str) -> None:
+    # Loads final results and subsets those files to a provide state excluding the Social Security Observer
+
+    from vivarium_census_prl_synth_pop.constants.metadata import US_STATE_ABBRV_MAP
+
+    abbrev_name_dict = {v: k for k, v in US_STATE_ABBRV_MAP.items()}
+    state_name = sanitize_location(abbrev_name_dict[state.upper()])
+    processed_results_dir = Path(processed_results_dir)
+    all_results = processed_results_dir / "usa"
+    state_dir = processed_results_dir / "states" / state_name
+    state_dir.mkdir(exist_ok=True, parents=True)
```

Comment:
This is creating the states directory, but it has the wrong permissions. You should use `utilities.build_output_dir()` to make sure it is created with the correct permissions.

Reply:
The child rhode island directory also has the wrong permissions